### PR TITLE
Del cleanup - uses a idempotent helper function

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 04a377c8172d4047cb69f19b92ac9cb0dc21c114:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2dae772f6a3daf3676e1a15aa9f562ef4055a34d:

Everything seems all right :smile: .
*CFME QE Bot*

File: artifactor/__init__.py [hunk 1]
```diff
@@ -284,8 +284,8 @@ def create_logger(logger_name, filename):
 
     """
     # If the logger already exists, destroy it
-    if logger_name in logging.root.manager.loggerDict:
-        del(logging.root.manager.loggerDict[logger_name])
+    # TODO: we should never need to poke into the logging internals
```
Comment (psav) [inline artifactor/__init__.py]: I think this is also present in the logger module as I think I stole the create_logger internals from there

File: utils/log.py [hunk 2]
```diff
@@ -340,8 +340,8 @@ def create_logger(logger_name, filename=None, max_file_size=None, max_backups=No
 
     """
     # If the logger already exists, destroy it
-    if logger_name in logging.root.manager.loggerDict:
-        del(logging.root.manager.loggerDict[logger_name])
+    # TODO: remove the need to destroy the logger
```
Comment (psav) [inline utils/log.py]: Ahhh I see it here ;)

**Comment by psav** [comment]
Does this still need to be WIP?

**Comment by psav** [comment]
Would like to get this merged, is it still needed to be WIP?

**Comment by psav** [comment]
Is this still WIP?

**Comment by dajoRH** [comment]
Lint report for commit fc3019460bcd90ab1e1ce5ee408cf0c8c972884c:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by psav** [comment]
Sorry @RonnyPfannschmidt I had to make this as unreviewed again - I see two different ways of clearing cache, one using ```del``` one using the ```clear_cached_property``` we should be consistent across the whole thing and use either one or the other unless there is a valid reason why not to.

**Comment by RonnyPfannschmidt** [comment]
the 2 in the test had a specific reason, ill try to figure what it was

**Comment by dajoRH** [comment]
Lint report for commit 3483b5a421ed16b504a234153df2b6003c0d5770:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by RonnyPfannschmidt** [comment]
the reason was a basic oversight

**Pull request merged**